gui:
  ros__parameters:
    combo_box:
      comboBox_line: 2
      comboBox_tool: 4
      comboBox_setting: 4
    tool_type:
      Tool0:
        comboBox_toolgeometry: 2
        comboBox_tooltype: 2
        comboBox_dir: 0
        sanding: 7
        tool_pressure: 10
        tool_length: 4
        tool_diameter: 5
        feed_rate: 6
        tool_rpm: 12000
      Tool1:
        comboBox_toolgeometry: 2
        comboBox_tooltype: 2
        comboBox_dir: 0
        sanding: 7
        tool_pressure: 10
        tool_length: 4
        tool_diameter: 5
        feed_rate: 6
        tool_rpm: 12000
      Tool2:
        comboBox_toolgeometry: 2
        comboBox_tooltype: 2
        comboBox_dir: 0
        sanding: 7
        tool_pressure: 10
        tool_length: 4
        tool_diameter: 5
        feed_rate: 6
        tool_rpm: 12000
      Tool3:
        comboBox_toolgeometry: 2
        comboBox_tooltype: 2
        comboBox_dir: 0
        sanding: 7
        tool_pressure: 10
        tool_length: 4
        tool_diameter: 5
        feed_rate: 6
        tool_rpm: 12000
      Tool4:
        comboBox_toolgeometry: 2
        comboBox_tooltype: 2
        comboBox_dir: 0
        sanding: 7
        tool_pressure: 10
        tool_length: 4
        tool_diameter: 5
        feed_rate: 6
        tool_rpm: 12000
    param_type:
      Param0:
        spindle_speed: 34
        grinding_force: 1000
        grinding_speed: 1000
        rotation: 1000
        linear: 4
      Param1:
        spindle_speed: 34
        grinding_force: 1000
        grinding_speed: 1000
        rotation: 1000
        linear: 4
      Param2:
        spindle_speed: 34
        grinding_force: 1000
        grinding_speed: 1000
        rotation: 1000
        linear: 4
      Param3:
        spindle_speed: 34
        grinding_force: 1000
        grinding_speed: 1000
        rotation: 1000
        linear: 4
      Param4:
        spindle_speed: 34
        grinding_force: 1000
        grinding_speed: 1000
        rotation: 1000
        linear: 4
    